### PR TITLE
Nodemailer Integration

### PR DESCRIPTION
**Pull request by ngambino0192**
This PR addresses several things:

1. Address a few build warnings related to tailwind and js linting
2. Fix Docker configuration to allow for hot reloading of both `frontend` and `backend`
3. Integrate `nodemailer` password reset flow

Something to keep in mind:

I have not created any seed data (and not really sure if I will). So, before you test the password reset flow, you will need to create a user account first, otherwise the server will throw a `500` error.

Lmk if you have any questions! I have a couple ideas for front end things that I think you might have fun building.

### CODE REVIEW

**Comment by melsoriano** [comment]
I'm having trouble logging in and sending an email recovery link. Looks like `.env` was modified. Can you secretly (not here in github) let me know what the correct values should be? Thanks!

File: frontend/src/App.js [hunk 1]
```diff
@@ -79,15 +82,20 @@ function App() {
   const [newslist, setNewslist] = useState([]);
   const [primaryTicker, setPrimaryTicker] = useState("AAPL");
   const [watchlist, setWatchlist] = useState([]);
-  const [bottomState, setBottomState] = useState(true);
+  const [, setBottomState] = useState(true);
```
Comment (melsoriano) [inline frontend/src/App.js:85]: What's the purpose of `setBottomState` if the state is not being used?

File: frontend/src/App.js [hunk 2]
```diff
@@ -79,15 +82,20 @@ function App() {
...
+  const [queryString] = useState(window.location.search);
+
+  const user = Cookies.get("user");
 
   useEffect(() => {
     const HTTP_OK = 200;
     const fetchData = async () => {
       let response = await fetch(
-        `http://localhost:6969/quote/${primaryTicker}`
+        `http://localhost:6969/api/markets/quote/${primaryTicker}`
```
Comment (melsoriano) [inline frontend/src/App.js:98]: What do you think about setting `https://localhost:6969/api/` as a `.env` variable called `API_HOST`? That way when this eventually gets deployed we only have to change the value in one spot.

**Review by melsoriano — APPROVED** [review]
Thank you for fixing the Docker config! 🐳 I left a couple of comments. Aside from those two comments, this PR is approved 🎉

File: frontend/src/App.js (hunk 2 above)
Comment (ngambino0192) [reply frontend/src/App.js]: Good idea, added in: https://github.com/ngambino0192/stocks/pull/11/commits/70cfe25b69530022d5e38f2c59f1ce4e73dee36f

File: frontend/src/App.js (hunk 1 above)
Comment (ngambino0192) [reply frontend/src/App.js]: Ahh, yeah we removed this didn't we. Cleaned up in: https://github.com/ngambino0192/stocks/pull/11/commits/1e680c51e34890fa1530ef09e5fb109317ec0edd

**Comment by ngambino0192** [comment]
Thanks for looking through that. I've addressed your comments plus a couple other things that I noticed while I was at it.

**Pull request merged**